A/Atomsk/build_tarballs.jl

```diff
@@ -3,29 +3,43 @@
 using BinaryBuilder, Pkg
 
 name = "Atomsk"
-version = v"0.11.2"
+version = v"0.12.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pierrehirel/atomsk.git", "3333858281e0ebd6279825b83ab871cf4d050a8d")
+    GitSource("https://github.com/pierrehirel/atomsk.git", "84f60a20c5b814fec03bffe1bccc3daaed0fc65d"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/atomsk/src/
-mkdir -p ${bindir}
+cd $WORKSPACE/srcdir/atomsk
+if [[ "$nbits" == 32 || "$target" == *apple* ]]; then
+    atomic_patch -p1 ../patches/atomsk_32.patch
+else
+    atomic_patch -p1 ../patches/atomsk_64.patch
+fi
+if [[ "$target" == *mingw* ]]; then
+    ln -s $prefix/bin/libatomsk.dll $prefix/lib/libatomsk.dll
+fi
```

Comment:
Why?

Reply:
I link the executable off of the lib that was just built and saved to `libdir`. But `libdir` is not in the `LD_LIBRARY_PATH` for `mingw` builds, so I link it there.

Reply:
> the lib that was just built and saved to `libdir`

That's where it's supposed to be, yes.

> But `libdir` is not in the `LD_LIBRARY_PATH`

What does that mean?  `LD_LIBRARY_PATH` has no role here (and is not a thing on Windows at all)

Reply:
I guess I mean that when I try to link it with `-latomsk`, it doesn't find the library unless I move it over to the `$prefix/lib` directory as well.

Reply:
There are other options besides creating useless links.

Reply:
Is there any option that you would prefer? Is there an environment variable that BinaryBuilder uses that I can set?

Reply:
I set LDFLAGS to include that directory in the search for libraries.